benchmark/blob/blob.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const common = require('../common.js');
+const { Blob } = require('buffer');
+
+const bench = common.createBenchmark(main, {
+  bytes: [0, 128, 512, 1024],
```

Comment:
```suggestion
  bytes: [0, 1024, 1024 ** 2],
```

Reply:
Thank you for the review. I just updated the pull request.